back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -19,7 +19,12 @@ def schema
   def json_forms_schema
     input = Idea.find params[:id]
     enabled_fields = IdeaCustomFieldsService.new(input.custom_form).enabled_fields
-    render json: JsonFormsService.new.input_ui_and_json_multiloc_schemas(enabled_fields, current_user)
+    schema = JsonFormsService.new.input_ui_and_json_multiloc_schemas(
+      enabled_fields,
+      current_user,
+      input.participation_context_on_creation.input_term
```

Comment:
Is this the correct input term? I would think that if an idea moves from phase 1 (with input term X) to phase 2 (with input term Y), then input term Y should be used?

Reply:
Good catch. I will look into it.